js/date-filter.js

```diff
@@ -12,6 +12,8 @@ require('jquery.inputmask/dist/inputmask/inputmask.numeric.extensions.js');
 function DateFilter(elm) {
   Filter.Filter.call(this, elm);
   this.validateInput = this.$elm.data('validate') || false;
+  this.$range = this.$elm.find('.range');
+  this.$grid = this.$elm.find('.date__range--grid');
```

Comment:
My bad, still getting used to BEM naming since I haven't used it before this project. Will change, thanks for the suggestion.